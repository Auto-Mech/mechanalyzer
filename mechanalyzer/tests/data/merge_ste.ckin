! This file simulates an AutoMech-calculated mechanism, w/stereospecific rxns
! Note: I'm assuming that all singly chiral (i.e., S/R) species have been 
! lumped together by this point


REACTIONS


C4ALK-808sWvZ + HO2 = C4CETH-RvEsWvBB0 + OH      1.00E+12     0    0
C4ALK-808sWvZ + HO2 = C4CETH-RvEsWvAB + OH       2.00E+12     0    0
C4ALK-808sWvE + HO2 = C4CETH-RvEsWvBB0 + OH      3.00E+12     0    0
C4ALK-808sWvE + HO2 = C4CETH-RvEsWvAB + OH       4.00E+12     0    0

! Two (nonsense) reactions to check when there is only a single
! stereoisomer in the *products*. They should be condensed to a single
! reaction (with the rates *averaged*)
H + HO2 = C4CETH-RvEsWvBB0 + OH      3.00E+12     0    0
H + HO2 = C4CETH-RvEsWvAB + OH       4.00E+12     0    0

! Two (nonsense) reactions to check when there is only a single
! stereoisomer in the *reactants*. They should be condensed to a single
! reaction (with the rates *added*)
C4ALK-808sWvZ + HO2 = H + OH      1.00E+12     0    0
C4ALK-808sWvE + HO2 = H + OH      2.00E+12     0    0

! Two non-stereo reactions. They should be unchanged
H + OH = H2O    1.00E+12    0   0
O + O = O2      1.00E+12    0   0

! A set of reactions with S/R only; should not affect the
! literature mechanism since they're singly chiral
RO2 = R + O2        1    0   0


END
