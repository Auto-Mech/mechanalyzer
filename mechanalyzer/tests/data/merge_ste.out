ELEMENTS

H
O
C

END


SPECIES

H                    ! SMILES: [H]                ChI: InChI=1S/H
O                    ! SMILES: [O]                ChI: InChI=1S/O
O2                   ! SMILES: O=O                ChI: InChI=1S/O2/c1-2
HO                   ! SMILES: [OH]               ChI: InChI=1S/HO/h1H
HO2                  ! SMILES: O[O]               ChI: InChI=1S/HO2/c1-2/h1H
H2O                  ! SMILES: O                  ChI: InChI=1S/H2O/h1H2
C4ALK-808sWvZ        ! SMILES: CC=CC              ChI: InChI=1S/C4H8/c1-3-4-2/h3-4H,1-2H3
C4ALK-808sWvE        ! SMILES: CC=CC              ChI: InChI=1S/C4H8/c1-3-4-2/h3-4H,1-2H3
C4CETH-RvEsWvAB      ! SMILES: CC1OC1C            ChI: InChI=1S/C4H8O/c1-3-4(2)5-3/h3-4H,1-2H3
C4CETH-RvEsWvBB0     ! SMILES: CC1OC1C            ChI: InChI=1S/C4H8O/c1-3-4(2)5-3/h3-4H,1-2H3
RO2                  ! SMILES: CCC(C)O[O]         ChI: InChI=1S/C4H9O2/c1-3-4(2)6-5/h4H,3H2,1-2H3
R                    ! SMILES: C[CH]CC            ChI: InChI=1S/C4H9/c1-3-4-2/h3H,4H2,1-2H3

END


THERMO
200.00    1000.00   5000.000

H                       C   4H   9O   2     G    200.00   1000.00 3000.00      1
 1.00000000E+00 3.40335819E-02-1.67583743E-05 4.00635161E-09-3.77164239E-13    2
-6.63687489E+03-1.86493221E+01 5.50455432E+00 3.84923631E-02 2.09295708E-06    3
-3.14168832E-08 1.60591535E-11-5.25830197E+03 5.02847289E+00                   4
O                       C   4H   9O   2     G    200.00   1000.00 3000.00      1
 9.82774981E+00 3.40335819E-02-1.67583743E-05 4.00635161E-09-3.77164239E-13    2
-6.63687489E+03-1.86493221E+01 5.50455432E+00 3.84923631E-02 2.09295708E-06    3
-3.14168832E-08 1.60591535E-11-5.25830197E+03 5.02847289E+00                   4
O2                      C   4H   9O   2     G    200.00   1000.00 3000.00      1
 2.00000000E+00 3.40335819E-02-1.67583743E-05 4.00635161E-09-3.77164239E-13    2
-6.63687489E+03-1.86493221E+01 5.50455432E+00 3.84923631E-02 2.09295708E-06    3
-3.14168832E-08 1.60591535E-11-5.25830197E+03 5.02847289E+00                   4
HO                      C   4H   8O   1     G    200.00   1000.00 3000.00      1
 3.59481396E+00 3.79952010E-02-1.93166856E-05 4.73431372E-09-4.54351344E-13    2
-1.88047079E+04 5.18969381E+00 4.32085317E+00-1.42723645E-04 1.10030584E-04    3
-1.46920106E-07 5.92646839E-11-1.76077436E+04 9.26039665E+00                   4
HO2                     C   4H   9O   2     G    200.00   1000.00 3000.00      1
 3.00000000E+00 3.40335819E-02-1.67583743E-05 4.00635161E-09-3.77164239E-13    2
-6.63687489E+03-1.86493221E+01 5.50455432E+00 3.84923631E-02 2.09295708E-06    3
-3.14168832E-08 1.60591535E-11-5.25830197E+03 5.02847289E+00                   4
H2O                     C   4H   9O   2     G    200.00   1000.00 3000.00      1
 4.00000000E+00 3.40335819E-02-1.67583743E-05 4.00635161E-09-3.77164239E-13    2
-6.63687489E+03-1.86493221E+01 5.50455432E+00 3.84923631E-02 2.09295708E-06    3
-3.14168832E-08 1.60591535E-11-5.25830197E+03 5.02847289E+00                   4
C4ALK-808sWvZ           C   4H   8          G    200.00   1000.00 3000.00      1
 2.04434790E+00 3.49270405E-02-1.74021987E-05 4.19332673E-09-3.96811612E-13    2
-3.92793440E+03 1.45603022E+01 6.23326980E+00-6.54731936E-03 9.54728725E-05    3
-1.16890205E-07 4.50970872E-11-3.83259685E+03-3.51218265E-01                   4
C4ALK-808sWvE           C   4H   8          G    200.00   1000.00 3000.00      1
 2.04434790E+00 3.49270405E-02-1.74021987E-05 4.19332673E-09-3.96811612E-13    2
-3.92793440E+03 1.45603022E+01 6.23326980E+00-6.54731936E-03 9.54728725E-05    3
-1.16890205E-07 4.50970872E-11-3.83259685E+03-3.51218265E-01                   4
C4CETH-RvEsWvAB         C   4H   8O   1     G    200.00   1000.00 3000.00      1
 3.75479487E+00 3.68492981E-02-1.85747644E-05 4.52149551E-09-4.31581834E-13    2
-1.98404075E+04 6.12315928E+00 5.31163766E+00 5.31516266E-03 7.94830710E-05    3
-1.06172173E-07 4.21815438E-11-1.91653357E+04 4.11869603E+00                   4
C4CETH-RvEsWvBB0        C   4H   8O   1     G    200.00   1000.00 3000.00      1
 3.75479487E+00 3.68492981E-02-1.85747644E-05 4.52149551E-09-4.31581834E-13    2
-1.98404075E+04 6.12315928E+00 5.31163766E+00 5.31516266E-03 7.94830710E-05    3
-1.06172173E-07 4.21815438E-11-1.91653357E+04 4.11869603E+00                   4
RO2                     C   4H   9O   2     G    200.00   1000.00 3000.00      1
 7.15325597E+00 3.84164962E-02-1.92514797E-05 4.66355946E-09-4.43390682E-13    2
-1.43381601E+04-6.11270866E+00 6.93923230E+00 1.42519726E-02 6.80197280E-05    3
-9.96905752E-08 4.10180835E-11-1.33360384E+04 3.13977230E-01                   4
R                       C   4H   9O   2     G    200.00   1000.00 3000.00      1
 5.00000000E+00 3.40335819E-02-1.67583743E-05 4.00635161E-09-3.77164239E-13    2
-6.63687489E+03-1.86493221E+01 5.50455432E+00 3.84923631E-02 2.09295708E-06    3
-3.14168832E-08 1.60591535E-11-5.25830197E+03 5.02847289E+00                   4

END


REACTIONS     CAL/MOLE     MOLES

H + HO2 = C4CETH-RvEsWvAB + HO                       1.750E+12     0.000        0

H + HO2 = C4CETH-RvEsWvBB0 + HO                      1.750E+12     0.000        0

C4ALK-808sWvZ + HO2 = C4CETH-RvEsWvAB + HO           2.500E+12     0.000        0

C4ALK-808sWvZ + HO2 = C4CETH-RvEsWvBB0 + HO          2.500E+12     0.000        0

C4ALK-808sWvE + HO2 = C4CETH-RvEsWvAB + HO           2.500E+12     0.000        0

C4ALK-808sWvE + HO2 = C4CETH-RvEsWvBB0 + HO          2.500E+12     0.000        0

R + O2 = RO2                                         1.000E+00     0.000        0

H + HO = H2O                                         1.000E+12     0.000        0

O + O = O2                                           1.000E+12     0.000        0

C4ALK-808sWvZ + HO2 = H + HO                         3.000E+12     0.000        0

C4ALK-808sWvE + HO2 = H + HO                         3.000E+12     0.000        0



END

